locales/en-US/strings.properties

```diff
@@ -5,25 +5,35 @@ bookmarked=Bookmarked
 synced=Synced from another device
```

Comment:
As noted elsewhere, a more descriptive identifier would better identify the usage of these strings, e.g., `section_header.top_sites`, `type_label.visited`